### PR TITLE
Unit test fixes for Windows

### PR DESCRIPTION
**Pull request by seriouz**
I've made some changes to the unit tests, so that they are all working with windows.
Almost all changes regarded the end-of-line problem between Windows and Linux/Mac.

### CODE REVIEW

**Comment by seriouz** [comment]
@dotnet-policy-service agree

**Comment by RikkiGibson** [comment]
Windows tests runs are timing out here.

Also seeing a warning in the test project: `/home/vsts/work/1/s/tests/Formatters/OrganizeImportsFormatterTests.cs(3,1): warning IDE0005: Using directive is unnecessary. [/home/vsts/work/1/s/tests/dotnet-format.UnitTests.csproj]`

**Comment by JoeRobich** [comment]
Not sure why these changes would be necessary.

**Review by sharwell — CHANGES_REQUESTED** [review]
In general this isn't the approach we'd want to go with for normalizing the tests. I'll need to review the variety of changes individually to discuss the resolution plan.

File: tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs [hunk 1]
```diff
@@ -86,21 +87,26 @@ void M()
 
         int count = 5;
 
+
+        count = 6;
```
Comment (sharwell) [inline tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs:91]: ❔ Why was this added?

File: tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs [hunk 2]
```diff
@@ -163,6 +169,9 @@ void M()
 
                 // Prefer using. IDISP017
                 ["dotnet_diagnostic.IDISP017.severity"] = "error",
+
+                // We expect new line as end_of_line in this test
+                ["end_of_line"] = EndOfLineFormatter.GetEndOfLineOption("\n"),
```
Comment (sharwell) [inline tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs:174]: ❔ Why was this added?

File: tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs [hunk 3]
```diff
@@ -210,6 +219,9 @@ void M()
...
```
Comment (sharwell) [inline tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs:224]: ❔ Why was this added? This test has `expectedCode` that relies on the environment newline, and the default `end_of_line` value would be the same. This test wouldn't have been failing before this change, provided the user on Windows checked out the code with the _required_ `core.autocrlf=true` configuration.

File: tests/CodeFormatterTests.cs [hunk 4]
```diff
@@ -587,9 +587,13 @@ public async Task GeneratorSolution_NoDiagnosticsReported_WhenNotIncludingGenera
...
-            const string ExpectedPublicApi = @"Greeter
-Greeter.Greet() -> void
-Greeter.Greeter() -> void";
+            var sb = new System.Text.StringBuilder();
+            sb.Append("Greeter");
+            sb.Append(Environment.NewLine);
+            sb.Append("Greeter.Greet() -> void");
+            sb.Append(Environment.NewLine);
+            sb.Append("Greeter.Greeter() -> void");
+            var expectedPublicApi = sb.ToString();
```
Comment (sharwell) [inline tests/CodeFormatterTests.cs:596]: ❔ Why was this changed?

File: tests/Formatters/OrganizeImportsFormatterTests.cs [hunk 5]
```diff
@@ -41,7 +41,7 @@ class C
 
             var editorConfig = new Dictionary<string, string>()
             {
-                ["end_of_line"] = EndOfLineFormatter.GetEndOfLineOption(Environment.NewLine),
+                ["end_of_line"] = EndOfLineFormatter.GetEndOfLineOption("\n"),
```
Comment (sharwell) [inline tests/Formatters/OrganizeImportsFormatterTests.cs:44]: ❔ Why was this changed? Applies to all instances of this in the file.

**Review by sharwell — COMMENTED** [review]
I suspect the author of this pull request might have not set `core.autocrlf=true` on a Windows machine, which would lead to expected failures. Prior to any fixes, please verify that the local configuration is correct.

**Comment by JoeRobich** [comment]
Closing as 9.x development has moved into the dotnet/sdk repo.

**Pull request closed without merging**